File: program_data/JavaProgramData/102/776.java
package <missing>;

public class GlobalMembers
{
	public static String sex = new String(new char[10]);
	public static double[] f = new double[40];
	public static double[] m = new double[40];
	public static int fsize;
	public static int msize;
	public static int desend(Object a, Object b)
	{
		if ((double)a > (double)b)
		{
			return -1;
		}
		else
		{
			return 1;
		}
	}
	public static int asend(Object a, Object b)
	{
		if ((double)a > (double)b)
		{
			return 1;
		}
		else
		{
			return -1;
		}
	}
	public static int Main()
	{
		int p;
		int i;
		int j;
		int k;
		double h;
		while (scanf("%d", p) != EOF)
		{
			fsize = msize = 0;
			for (i = 0;i < p;i++)
			{
				sex = ConsoleInput.readToWhiteSpace(true).charAt(0);
				h = Double.parseDouble(ConsoleInput.readToWhiteSpace(true));
				if (strcmp(sex,"male") == 0)
				{
					m[msize] = h;
					msize++;
				}
				else
				{
					f[fsize] = h;
					fsize++;
				}
			}
			qsort(f,fsize,(Double.SIZE / Byte.SIZE),desend);
			qsort(m,msize,(Double.SIZE / Byte.SIZE),asend);
			for (i = 0;i < msize;i++)
			{
				System.out.printf("%.2lf ",m[i]);
			}
			for (i = 0;i < fsize-1;i++)
			{
				System.out.printf("%.2lf ",f[i]);
			}
			System.out.printf("%.2lf\n",f[fsize-1]);
		}
		return 0;
	}
}

File: program_data/ProgramData/83/776.cpp
int main()
{
	int n,i,z=0;
	int a[10];
	int b[10];
	float c[10],g,y=0,t[10]={0};
	scanf("%d",&n);
	for(i=0;i<n;i++)
	{
		scanf("%d",&a[i]);
                  z=z+a[i];
	}
	for(i=0;i<n;i++)
	{
                  scanf("%d",&b[i]);
		if(90<=b[i]&&b[i]<=100)
			{c[i]=4.0;}
		else if(85<=b[i]&&b[i]<=89)
			{c[i]=3.7;}
		else if(82<=b[i]&&b[i]<=84)
			{c[i]=3.3;}
		else if(78<=b[i]&&b[i]<=81)
			{c[i]=3.0;}
		else if(75<=b[i]&&b[i]<=77)
			{c[i]=2.7;}
		else if(72<=b[i]&&b[i]<=74)
			{c[i]=2.3;}
		else if(68<=b[i]&&b[i]<=71)
			{c[i]=2.0;}
		else if(64<=b[i]&&b[i]<=67)
			{c[i]=1.5;}
		else if(60<=b[i]&&b[i]<=63)
			{c[i]=1.0;}
		else if(b[i]<60)
			{c[i]=0;}
		t[i]=c[i]*a[i];
	         y=y+t[i];
	}
	g=y/z;
	printf("%.2f",g);
	return 0;
}
